groups/live-windows-security-groups/smart-vault-sg.tf

```diff
@@ -0,0 +1,181 @@
+# ------------------------------------------------------------------------------
+# Smart Vault 1 Server - ch-smartv1-12 Security Group and rules
+# ------------------------------------------------------------------------------
+module "smart_vault_2_server_security_group" {
```

Comment:
naming of group doesnt match rules

Reply:
# ------------------------------------------------------------------------------
# Smart Vault 1 Server - ch-smartv1-12 Security Group and rules
# ------------------------------------------------------------------------------
module "smart_vault_1_server_security_group" {
  source  = "terraform-aws-modules/security-group/aws"
  version = "~> 3.0"